core/node/node.go

```diff
@@ -54,8 +55,15 @@ func NewNode(
 		return dialogEstablisher.EstablishDialog(providerID, contact)
 	}
 
-	promiseIssuerFactory := func(dialog communication.Dialog) connection.PromiseIssuer {
-		return noop.NewPromiseIssuer(dialog)
+	promiseIssuerFactory := func(issuerID identity.Identity, dialog communication.Dialog) connection.PromiseIssuer {
+		return &noop.PromiseIssuer{
+			IssuerID: issuerID,
+			Dialog:   dialog,
+			Signer:   signerFactory(issuerID),
+			Amount: money.Money{
```

Comment:
You are getting `issuer.Start(proposal)` and that data should be enough to create promises, because it holds `primise.PaymentMethod`.
Probably we need something as `NoopPaymentMethod`

Later:
We agreed to create `PromisePolicy` interface , which answers to political questions. We can implement it with later PR.

Reply:
Changed it to getting `Amount` from the `promise`